modules/op-release-notes-1-8.adoc

```diff
@@ -20,6 +20,44 @@ With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift
 
 include::snippets/technology-preview.adoc[]
 
+=== Adding Git repositories by using webhooks
+
+With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible.
+// (link:https://issues.redhat.com//browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+=== Running multiple PipelineRun objects
+
+Previously, as a developer, you could run only one `PipelineRun` object from your `.tekton` directory. With this update, you can run all the `PipelineRun` resources in your `.tekton` directory that match specified events by using the *Developer* perspective of the {product-title} web console. The console displays the status and reports of the runs. If you are using WebVCS, the reports display comments.
+// (link:https://issues.redhat.com//browse/SRVKP-1781[SRVKP-1781 Multiple Pipelines runs match on events])
+// Chmouel Boudjnah
+
+=== Installing Tekton Hub with better defaults
```

Comment:
Suggestion: "Tekton Hub installation uses default configurations"

"With this update you can install a Tekton Hub with minimal configurations required so that you can ..."

Reply:
Updated. Thanks.